app/controllers/users/webauthn_setup_controller.rb

```diff
@@ -193,7 +192,13 @@ def process_invalid_webauthn(form)
       elsif form.platform_authenticator?
         flash[:error] = t('errors.webauthn_platform_setup.general_error')
       else
-        flash[:error] = t('errors.webauthn_setup.general_error')
+        flash[:error] = t(
+          'errors.webauthn_setup.general_error_html',
+          link_html: view_context.link_to(
+            t('errors.webauthn_setup.additional_methods_link'),
+            authentication_methods_setup_path,
+          ),
+        )
```

Comment:
When does this error happen in the real-world? It seems like it might be susceptible to the same conditions we'd want to apply in `WebauthnVisitForm#webauthn_general_error` with omitting the link for errors when adding a security key from the account dashboard.

Reply:
(discussed elsewhere)